src/calculateRentalCost.js

```diff
@@ -28,7 +28,27 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const shortTerm = 3;
+  const shortTermDiscount = 20;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const pricePerDay = 40;
+
+  let totalPrice = pricePerDay * days;
+
+  if (days >= longTerm) {
+    totalPrice -= longTermDiscount;
+
+    return totalPrice;
```

Comment:
No need to return `totalPrice` here, you will do it at line `51` in your code

Reply:
@adioskv She can't remove this return because she uses let with totalPrice and this condition `days >= shortTerm` will always overwrite this one `if (days >= longTerm)` so she needs an early return